src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -44,48 +44,48 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Should this be PUT instead of GET? GET generally means this servlet is going to return some data. PUT generally means this servlet is going to update something (in our case, the database) in an idempotent manner (as opposed to POST). 

https://www.w3schools.com/tags/ref_httpmethods.asp

Reply:
Yes -- I had it as a GET because the database isn't set up yet, and I wanted to be able to view the output at the /election URL for debugging purposes. Let me add the appropriate TODOs indicating this right now.